docs/content/users/usage/commands.md

```diff
@@ -596,8 +611,16 @@ ddev get /path/to/package
 
 # Copy an add-on from a tarball in another directory
 ddev get /path/to/tarball.tar.gz
+
+# View installed add-ons
+ddev get --installed
+
+# Remove an add-on
+ddev get --remove redis
 ```
 
+In general, you can `ddev get` an add-on multiple times without doing any damage; Updating an add-on can be done with just `ddev get <add-on-name>`. If you have changed an add-on file and removed the `#ddev-generated` marker in the file, that file will not be touched and DDEV will let you know about it.
```

Comment:
Run commands, don’t treat `ddev get` as a verb where we’re explaining what it does. Fix punctuation. Avoid “just” again.

```suggestion
In general, you can run `ddev get` multiple times without doing any damage. Updating an add-on can be done by running `ddev get <add-on-name>`. If you have changed an add-on file and removed the `#ddev-generated` marker in the file, that file will not be touched and DDEV will let you know about it.
```